_editorial_policies/04_publication_ethics.md

```diff
@@ -0,0 +1,51 @@
+- In submitting a manuscript for publication, an author should inform the editor of related manuscripts that the author has under editorial consideration or in press. Copies of those manuscripts should be supplied to the editor, and the relationships of such manuscripts to the one submitted should be indicated.					
+- An author should identify the source of all information quoted or offered, except that which is common knowledge. Information obtained privately, as in conversation, correspondence, or discussion with third parties, should not be used or reported in the author’s work without explicit permission from the investigator with whom the information originated. Information obtained in the course of confidential services, such as refereeing manuscripts or grant applications, should be treated similarly	
+- Material submitted to LiveCoMS may sometimes justify criticism, even severe criticism, of the work of another scientist. However, in no case is personal criticism considered to be appropriate.
+- Plagiarism is serious ethical breach, and the consequences of plagarism are discussed in the LiveCoMS bylaws. With continually updated reviews published in LiveCoMS and the possibility of including some material from previously published sources, the LiveCoMS model raises some additional questions as to what constitutes plagiarism. We discuss this further in the “Plagarism and Prior Publication” section of the Author Instructions.
+- Authors are encouraged to recommend appropriate reviewers during submission of a manuscript. However, they should not suggest reviewers with whom they have an actual, perceived, or potential conflict of interest. Moreover, authors should not suggest reviewers with whom they share a personal or professional connection if there is potential for that relationship to introduce bias into the evaluation of the manuscript. 
+- Submitting authors must ensure that the listed co-authors of a paper should be all those persons who have made significant scientific contributions to the work reported and who share responsibility and accountability for the results. Authors should appropriately recognize the contributions of technical staff and data professionals. Other contributions should be indicated in a footnote or an “Acknowledgments” section. An administrative relationship to the work does not of itself qualify a person for co-authorship (but occasionally it may be appropriate to acknowledge major administrative assistance). Deceased persons who meet the criterion for inclusion as co-authors should be so included, with a footnote reporting date of death. No fictitious name should be listed as an author or coauthor. The author who submits a manuscript for publication accepts the responsibility of having included as co-authors all persons appropriate and none inappropriate. The submitting author should have sent each living co-author a draft copy of the manuscript and have obtained the co-author’s assent to co-authorship of it.
```

Comment:
Don't we somewhere have a policy on author REMOVAL from subsequent versions of an article? I imagine in the case of a dead author they should likely be removed from subsequent versions if the authors list changes; an honorary authorship of new/distinct versions is probably not warranted.

Reply:
I added a comment about authorship with living documents and a link to the relevant text.